### PR TITLE
Iz style ws list view

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: src/pages/workspaces/workspace/Container.js [hunk 1]
```diff
@@ -23,17 +26,19 @@ const tabActiveStyle = _.defaults({
...
 const tabComponents = {
   dashboard: WorkspaceDashboard,
   notebooks: WorkspaceNotebooks,
   data: WorkspaceData,
-  jobs: () => div('when we get to it'),
-  history: () => div('okay, this is _data_ history, whatever that means'),
-  tools: () => div('These are totally Not Method Configs')
+  jobs: () => div('Job manager goes here'),
+  history: () => div('Data history goes here'),
+  tools: () => div('Pipelines from Dockstore go here')
```
Comment (dmohs) [inline src/pages/workspaces/workspace/Container.js:41]: Nice ;)

File: src/components/ShowOnClick.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    }
+  }
+
+  setVisibility = (visible) => {
+    this.setState({ hidden: !visible })
+  }
+
+  listenForEscape = (e) => {
+    if (e.key === 'Escape') {
+      window.removeEventListener('keydown', this.listenForEscape)
```
Comment (dmohs) [inline src/components/ShowOnClick.js:28]: What happens if this component gets unmounted while open? Will the event listener still get removed?

File: src/pages/workspaces/List.js [hunk 3]
```diff
@@ -23,24 +26,159 @@ const WorkspaceList = hh(class WorkspaceList extends Component {
...
+            height: '1.5rem', width: '1.5rem', borderRadius: '1.5rem', lineHeight: '1.5rem',
+            textAlign: 'center', backgroundColor: 'white',
+            boxShadow: Style.contextMenuShadow, marginBottom: '0.5rem'
+          }
+        }, [icon('bars')])
+      },
+      [
+        div({ style: { position: 'absolute', left: '2rem', top: '0.75rem' } }, [
+          contextMenu([
+            [{ onClick: () => Nav.goToPath('workspace', namespace, name) }, 'Open'],
```
Comment (dmohs) [inline src/pages/workspaces/List.js]: Command-click doesn't open this in a new tab.

File: src/components/Modal.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+
+/**
+ * @param onDismiss
+ * @param title
+ * @param children
+ * @param showCancel=true
+ * @param okButton
+ */
+export default hh(class Modal extends Component {
```
Comment (dmohs) [inline src/components/Modal.js:13]: Doesn't appear to be used, so hard to test for code review. In the future, I'm totally fine with having a button who's sole purpose is to try out a control like this. We could enable it via feature flag if we want prod to be clean.

**Review by dmohs — APPROVED** [review]

File: config/prod.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
   "googleClientId": "424501080111-b3bt4p1vo4gbo4m0573nvi4d49784bp8.apps.googleusercontent.com",
-  "leoUrlRoot": "https://leonardo.dsde-prod.broadinstitute.org",
+  "leoUrlRoot": "https://notebooks.firecloud.org",
```
Comment (dmohs) [inline config/prod.json]: This should be a separate PR. That would allow me to merge the fix to prod while we discuss this PR.

**Comment by zarsky-broad** [comment]
@dmohs ready for re-review

File: src/components/Modal.js (hunk 4 above)
Comment (zarsky-broad) [reply src/components/Modal.js:13]: Now used

File: src/components/ShowOnClick.js (hunk 2 above)
Comment (zarsky-broad) [reply src/components/ShowOnClick.js:28]: Added `willUnmount` to handle this

File: src/pages/workspaces/workspace/Dashboard.js [hunk 6]
```diff
@@ -24,16 +26,29 @@ export default hh(class WorkspaceDashboard extends Component {
...
         spinner({ style: { marginTop: '1rem' } }) :
       div({ style: { margin: '1rem' } }, [
+        modal ? Modal({
+          onDismiss: () => this.setState({ modal: false }),
+          title: 'Workspace Info',
+          showCancel: false,
+          okButton: buttonPrimary({ onClick: () => this.setState({ modal: false }) }, 'Done')
+        }, [
+          div({ style: { whiteSpace: 'pre', overflow: 'auto', padding: '1rem' } },
+            JSON.stringify(workspace, null, 2))
```
Comment (dmohs) [inline src/pages/workspaces/workspace/Dashboard.js:43]: Great example! I see this minor issue: https://www.useloom.com/share/e32a7a430cf4457e8becc43ba9e5ca77 Feel free to add an issue to address in a follow-up if this is blocking other work.

Comment (zarsky-broad) [reply src/pages/workspaces/workspace/Dashboard.js:43]: Created #60
